pkg/ebpf/common.go

```diff
@@ -167,3 +167,20 @@ func isUbuntu(platform string) bool {
 func isLinuxAWSUbuntu(platform string) bool {
 	return strings.Contains(strings.ToLower(platform), "aws") && isUbuntu(platform)
 }
+
+func isCentOS(platform string) bool {
```

Comment:
Can you add some tests for all these based on real samples you've been able to obtain?

Reply:
Tests are added.